virtme_ng/run.py

```diff
@@ -352,6 +352,14 @@ def make_parser():
         help="Enable network access: user, bridge(=<br>), loop",
     )
 
+    parser.add_argument(
+        "--net-mac-address",
+        "-N",
```

Comment:
Do we really need a short option here? Can we just use the long option like the virtme-run backend?

My concern is that we may want to use `-N` for something more "common" in the future (I'm not sure how many people will need to change the mac addr) and if we reserve this option here, then it's problematic to change it later.

Reply:
Good point, I will remove this short option!